src/KafkaFlow/Batching/BatchingExtensions.cs

```diff
@@ -1,4 +1,4 @@
-namespace KafkaFlow.BatchConsume
+namespace KafkaFlow.Batching
```

Comment:
I think it's a trade-off between ease of use and project organization

Reply:
As part of the setup process, we have placed the extension methods in the root KafkaFlow namespace. This simplifies their usage and ensures that there are fewer namespaces to include in the application's bootstrap file. @jose-sousa-8, it would be appreciated if you could also change the namespace of the Serialization and Compression extensions.

Reply:
Done here https://github.com/Farfetch/kafkaflow/pull/435/commits/2a3d7b3489453a366b6a673ca3973375451b33d0